onnxscript/tests/function_libs/torch_lib/ops_test_common.py

```diff
@@ -292,7 +292,7 @@ def convert_kwargs_for_onnx(kwargs: dict[str, Any]) -> dict[str, Any]:
         if key == "dtype":
             value = TORCH_TYPE_TO_ONNX[value]
         if isinstance(value, torch.Tensor):
-            value = np.array(value)
+            value = np.array(value.cpu())
```

Comment:
Did this break anything, or it's just safer to have?

Reply:
I think safer since its noop on cpu. Without it I'm observing test failure on local env with cuda.